README.md

```diff
@@ -32,8 +34,70 @@ To contribute to Stacks documentation or its CSS library, you’ll need to build
 
 Having trouble getting these steps to work? Open [an issue](https://github.com/StackExchange/Stacks/issues/new) with a `setup` label.
 
+## Linting Stacks
+
+Run all lint suites by running:
+```sh
+npm run lint
+```
+
+Lint the styles (stylelint) by running:
+```sh
+npm run lint:css
+```
+Lint the typescript source code (eslint) via running:
+```sh
+npm run lint:ts
+```
+Lint the source code format (prettier) via running:
+```sh
+npm run lint:format
+```
 ## Testing Stacks
-Stacks has implemented visual regression testing with [Backstop](https://github.com/garris/BackstopJS). To test if your new feature introduces visual regressions, run `npm run test` in a new Terminal window while Stacks is running. After the tests have run, a new browser window with any regressions will show. If the regressions are desired, you can run `npm run approve` to establish the new baseline.
+
+Run all test suites by running:
+```sh
+npm test
+```
+### Unit/Component Tests
+
+Unit/Component tests are written with [DOM Testing Library](https://testing-library.com/docs/dom-testing-library/intro).
+Please follow the library's principles and documentation to write tests.
+
+Stacks uses [Web Test Runner](https://modern-web.dev/docs/test-runner/overview/) and [Playwright](https://modern-web.dev/docs/test-runner/browser-launchers/playwright/) to run tests in a real browser context.
+
+Execute the unit/component tests suite by running:
+```sh
+npm run test:unit
+```
+or if you prefer watch mode run:
+```sh
+npm run test:unit:watch
+```
+
+### Visual Regression Tests
+
+**Prerequisit:** 
+In order to pull and upload baseline images you need to have `git lfs` installed in your local machine. Please follow [this guide](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage) to install that.
```

Comment:
We could add that command to the the `prepare` script in `package.json`. We do something similar in Stacks-Editor to initialize Husky locally: https://github.com/StackExchange/Stacks-Editor/blob/main/package.json#L28

Reply:
`git-lfs` is something we will be installing **globally** in the user machine. 
Personally I'd rather have the user opt-in explicitly for that one-off installation.
Since the tool is only necessary for people who wants to run (visual) tests we might add a `pretest` warning if `git-lfs` is not in path - we could further optimise it with a console prompt (y/n) style. I will add the warning for now.
Thanks for the idea! 🙂

Reply:
@b-kelly I'd like to postpone this bit if it's ok with you as a "nice to have" we can do any time. While I was setting up my windows environment to debug the issue you experienced I realized git installation from chocolatey comes with `lfs` by default. I think `lfs` won't cause a lot of friction but if you have something specific in mind feel free to go ahead and update this PR.

Reply:
I'm ok with pushing this out. It wasn't so much a hard requirement, but a request for thoughts / comments. I'm a little jumpy about committing images to the repo (because of prior mistakes made in this repo), but that's not really based in reality.